packages/compass-schema-validation/src/stores/store.spec.js

```diff
@@ -113,27 +117,36 @@ describe('Schema Validation Store', () => {
       });
     });
 
-    context('when the action is VALIDATION_FETCHED', () => {
+    context('when the action is fetch sample documents', function() {
+      it('updates the sample document loading in state', function(done) {
+        expect(store.getState().sampleDocuments.isLoading).to.equal(false);
+        const unsubscribe = store.subscribe(() => {
+          unsubscribe();
+          expect(store.getState().sampleDocuments.isLoading).to.equal(true);
+          done();
+        });
+        store.dispatch(fetchSampleDocuments({ matching: {}, notmatching: {}}));
+      });
+    });
+
+    context('when the action is VALIDATION_FETCHED', function() {
       const validation = {
         validator: { name: { $type: 4 } },
         validationAction: 'warn',
         validationLevel: 'moderate'
       };
 
-      it('updates the isLoading in state', () => {
-        const isLoading = true;
-
-        it('updates the validationAction in state', (done) => {
```

Comment:
It looked like this test was never run (`it` inside `it`). I broke it out and updated the test to test what I think it was going for.